.github/actions/pr-title-checker.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
```

Comment:
We need to change the GitHub action used here to verify the PR title
to either test the PR title of this repo with the new solution so that we can ensure that works